components/UserPaymentMethod.vue

```diff
@@ -130,4 +200,23 @@ div {
 .plus-circle-icon {
   margin-right: calc(var(--spacer-2xs) * 5);
 }
+
+.action-buttons {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: calc(var(--spacer-xl) / 8);
+
+  &__cancel {
+    background-color: var(--_c-light-primary);
+    border: 1px solid var(--_c-gray-middle);
+    color: var(--c-black);
+  }
+
+  &__update {
+    border: none;
+    background-color: var(--_c-green-primary);
```

Comment:
Do we need to provide background color for both Cancel and Update in the component level? Can we handle it by just adding predefined classes like  class="color-light" on SfButton?